### PR TITLE
ISSUE-11 - SwiftFormat config for project + Header + Attribution

### PR DESCRIPTION
**Pull request by AdonisCodes**
# Reason

Currently we aren't giving the original developer of this package enough credit. We also don't have any consistency in the codebase due to the lack of formatting rules. 

# Tech Details List

- Initializes + creates  `.swiftformat` & `.swift-version` files
- Adds a descriptive header for all package files
- Formats all documents

# Review Checklist
- [x] Properly Typed All Symbols
- [x] Updated All documentation that the code touched
- [x] Testing Fresh Deployment works with a singular command
- [x] Tests cover all discovered edge cases
- [x] Handles all errors gracefully, ensuring unexpected errors are tracked & monitored
- [x] Code Structure Correctness
- [x] Cleanup of temporary/noise/testing files
- [x] Load Testing: Ensure the code handles expected traffic efficiently

### CODE REVIEW

**Comment by vimscientist69** [comment]
@AdonisCodes, is this pull request ready to be merged? In the future, can you assign me as a reviewer when you think it is ready to be reviewed (after you've reviewed it first yourself)?

File: Sources/SwiftWebDriver/API/Request/DevTools/PostExecuteAsyncRequest.swift [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+// All rights reserved.
+//
+// This package is freely distributable under the MIT license.
+// This Package is a modified fork of https://github.com/ashi-psn/SwiftWebDriver.
+
 import AsyncHTTPClient
-import NIOHTTP1
+import Foundation
 import NIO
+import NIOHTTP1
```
Comment (vimscientist69) [inline Sources/SwiftWebDriver/API/Request/DevTools/PostExecuteAsyncRequest.swift:12]: Why import this?

File: Sources/SwiftWebDriver/API/Request/Elements/GetElementAttributeRequest.swift [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+// Copyright (c) 2025 GetAutomaApp
+// All source code and related assets are the property of GetAutomaApp.
+// All rights reserved.
+//
+// This package is freely distributable under the MIT license.
+// This Package is a modified fork of https://github.com/ashi-psn/SwiftWebDriver.
+
 import AsyncHTTPClient
-import NIOHTTP1
+import Foundation
```
Comment (vimscientist69) [inline Sources/SwiftWebDriver/API/Request/Elements/GetElementAttributeRequest.swift:10]: Why import Foundation? The code seemed to work beforehand without importing it. Maybe this is just the formatter.

File: Sources/SwiftWebDriver/API/Response/Elements/GetSessionActiveElementResponse.swift [hunk 3]
```diff
@@ -24,7 +31,7 @@ public struct GetSessionActiveElementResponse: ResponseType {
         }
 
         init?(intValue: Int) {
-            self.stringValue = "\(intValue)"
+            stringValue = "\(intValue)"
```
Comment (vimscientist69) [inline Sources/SwiftWebDriver/API/Response/Elements/GetSessionActiveElementResponse.swift:34]: Should we use String(intValue) in scenarios like this?

**Review by vimscientist69 — APPROVED** [review]
Good overall. Respond to my comments and make any changes mentioned from them (if any), then merge.

**Comment by vimscientist69** [comment]
We should create a `package.json` file with install/setup scripts (for example install swiftlint and swiftformat, format all code) , etc for Github workflow files we'll create in the future. Create a ticket for that (feel free to assign me to it).

File: Sources/SwiftWebDriver/API/Request/DevTools/PostExecuteAsyncRequest.swift (hunk 1 above)
Comment (AdonisCodes) [reply Sources/SwiftWebDriver/API/Request/DevTools/PostExecuteAsyncRequest.swift:12]: Formatter just switched around imports

File: Sources/SwiftWebDriver/API/Request/Elements/GetElementAttributeRequest.swift (hunk 2 above)
Comment (AdonisCodes) [reply Sources/SwiftWebDriver/API/Request/Elements/GetElementAttributeRequest.swift:10]: formatter import reorder
